lib/sycamore/sycamore/connectors/file/file_writer_ray.py

```diff
@@ -25,6 +25,8 @@ def __init__(
     ):
         (paths, self._filesystem) = _resolve_paths_and_filesystem(path, filesystem)
         self._root = paths[0]
+        if self._root == "":
```

Comment:
so sorry, but I think you need to check for s3 uris here too, right?

Reply:
Assuming we are using `urlparse(url).scheme == "s3"` to decide if its an s3 uri or not it shouldn't matter because both "" and "./" return False

Unless I'm missing something, this feels redundant!

Reply:
fair enough. ok!